client/post-editor/post-editor.jsx

```diff
@@ -206,9 +218,21 @@ const PostEditor = React.createClass( {
 								site={ site }
 								type={ this.props.type }
 							/>
+							<div className="editor__site">
```

Comment:
`className` should be `post-editor__site` in accordance with our CSS naming guidelines (though, to be fair, the component itself should probably be in its own directory, not named `post-editor.jsx`)